torch/utils/serialization/config.py

```diff
@@ -0,0 +1,20 @@
+import sys
+from typing import Optional as _Optional
+
+from torch.serialization import LoadEndianness as _LoadEndianess
```

Comment:
To avoid the circular import that forces you to do in-function import in the other file, should this enum be defined here and just imported in the the serialization.py file?

Reply:
Ok, let's keep it as is then.